specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-02-01/management.JSON

```diff
@@ -0,0 +1,2000 @@
+        }
+      }
+    },
+    "CreateManagementGroupChildInfo": {
+      "description": "The child information of a management group used during creation.",
+      "properties": {
+        "type": {
+          "title": "The type of child resource.",
+          "readOnly": true,
+          "$ref": "#/definitions/ManagementGroupChildType",
+          "description": "The fully qualified resource type which includes provider namespace (e.g. Microsoft.Management/managementGroups)"
+        },
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The fully qualified ID for the child resource (management group or subscription).  For example, /providers/Microsoft.Management/managementGroups/0000000-0000-0000-0000-000000000000"
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the child entity."
+        },
+        "displayName": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The friendly name of the child resource."
+        },
+        "roles": {
+          "description": "The roles definitions associated with the management group.",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "children": {
+          "description": "The list of children.",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CreateManagementGroupChildInfo"
+          }
+        }
+      }
+    },
+    "CreateParentGroupInfo": {
+      "description": "(Optional) The ID of the parent management group used during creation.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The fully qualified ID for the parent management group.  For example, /providers/Microsoft.Management/managementGroups/0000000-0000-0000-0000-000000000000"
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the parent management group"
+        },
+        "displayName": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The friendly name of the parent management group."
+        }
+      }
+    },
+    "Permissions": {
+      "type": "string",
+      "enum": [
+        "noaccess",
+        "view",
+        "edit",
+        "delete"
+      ],
+      "description": "The users specific permissions to this item."
+    },
+    "CheckNameAvailabilityRequest": {
+      "description": "Management group name availability check parameters.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "the name to check for availability"
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "/providers/Microsoft.Management/managementGroup"
```

Comment:
Why are you changing this enum value?

the ARM resource type name doesn't have '/providers/' in it and could cause confusion.

Reply:
Removing that too, was added by mistake.